### ISSUE TITLE
Quote mark in video file name prevents playback

### ISSUE THREAD

**Issue by dickp** [MediaBrowser/Emby#3747]
Filename special character escaping fails to escape " in names.  For example, from my server's attached ffmpeg-transcode log it clearly shows the ffmpeg process trying to open a file where the name is truncated - and obviously failing.  Playback then fails with the slightly misleading error "No compatible streams are currently available. Please try again later or contact your system administrator for details."

[ffmpeg-transcode-7e91dc05-3927-4130-b077-f9d9ec4e0084_1.txt](https://github.com/MediaBrowser/Emby/files/9893366/ffmpeg-transcode-7e91dc05-3927-4130-b077-f9d9ec4e0084_1.txt)

**Comment by softworkz** [comment]
How about renaming the file..?

**Comment by softworkz** [comment]
But more seriously: This has been implemented and verified to be working. 
I just simulated the whole procedure with your filename and it ended up being escaped in the ffmpeg log.
So I'm not sure why it wouldn't be working at your side.

We need to start with a simple test. Can you please:

- Create a library on a local disk - for example, create a folder `/var/packages/EmbyServer/var/testlib`
- Copy a small and simple video file (with a short name and no special chars) into that folder
- Add this folder as new library in Emby
- Play the file, lower the quality to make sure it transcodes
- Now, rename the file in a way that you put a double quote (") somewhere in the middle of the file name.
- Wait 1-2 min until Emby has refreshed
- Play again and see whether it works, if not, post the ffmpeg log

Thanks

**Comment by cayars** [comment]
It's encoded wrong and not working for me on any Linux based platform. I renamed A Beautiful Mind (2001).mkv to /A Beautiful "Mind (2001)".mkv so there was a set of double quotes.  It' fails to playback. 
Tried it with a single " as well and doesn't work.
It's not correctly encoded from looking at the log. I tried a test with two double quotes in a file name and it matches what's shown here: Debug SqliteItemRepository: GetitemById Movie 6880767 /volume1/Media/Movies/A/A Beautiful Mind (2001)/A Beautiful "Mind (2001)".mkv
[embyserver (32).txt](https://github.com/MediaBrowser/Emby/files/11233792/embyserver.32.txt)
